examples/kotlin-diktat/save.toml

```diff
@@ -1,5 +1,5 @@
 [general]
-    execCmd = "java -jar ktlint-old --disabled_rules=standard -R diktat-old.jar"
+    execCmd = "java -jar ktlint --disabled_rules=standard -R diktat.jar"
```

Comment:
we release diktat in format `diktat-*.jar`, maybe change it to `java -jar ktlint --disabled_rules=standard -R diktat-*.jar`?